data/json/effects.json

```diff
@@ -2073,6 +2073,52 @@
     "rating": "bad",
     "base_mods": { "vomit_chance": [ 500 ] }
   },
+  {
+    "type": "effect_type",
+    "id": "bile_stink",
+    "name": [ "Bile Stink" ],
+    "desc": [ "You're splattered with a greasy layer of liquified putrescence." ],
+    "apply_message": "You're covered in a foul smelling fluid!",
+    "rating": "bad",
+    "resist_traits": [ "FLOWERS", "ROSEBUDS", "SAPROVORE", "EATDEAD" ],
+    "base_mods": { "vomit_chance": [ 500, 2000 ] },
+    "enchantments": [
+      {
+        "values": [
```

Comment:
Actially just drop all the social modifiers.

Reply:
It was all based on `skunk_spray`.